### PR TITLE
BAT-7558 - ME 0.13 and Django update

### PR DESCRIPTION
**Pull request by asiebert**
:warning: → superseded by #4

### CODE REVIEW

File: tastypie_mongoengine/resources.py [hunk 1]
```diff
@@ -97,7 +97,8 @@ def __new__(cls, name, bases, attrs):
                     del(new_class.base_fields[field_name])
             if field_name in new_class.declared_fields:
                 continue
-            if len(include_fields) and field_name not in include_fields:
+            # Meta.fields is None by default
+            if include_fields and len(include_fields) and field_name not in include_fields:
```
Comment (m-vdb) [inline tastypie_mongoengine/resources.py]: you can just remove the `len(include_fields)`

**Review by m-vdb — COMMENTED** [review]
...

File: tastypie_mongoengine/resources.py (hunk 1 above)
Comment (asiebert) [reply tastypie_mongoengine/resources.py]: 8a7d7fa9b1f78b511dc789e6a4718370b69d46e0 :white_check_mark:

**Comment by asiebert** [comment]
Fixed branch name in #4

**Pull request closed without merging**